network-api/networkapi/wagtailpages/views.py

```diff
@@ -34,4 +34,16 @@ def redirect_to_initiatives(request, subpath):
     if request.META['QUERY_STRING']:
         query_string = f'?{request.META["QUERY_STRING"]}'
 
-    return redirect(f'/{request.LANGUAGE_CODE}/initiatives/{subpath}{query_string}')
+    return redirect(f'/{request.LANGUAGE_CODE}/{destination_path}/{subpath}{query_string}')
+
+
+def redirect_to_whoweare(request, subpath):
```

Comment:
It feels like this code should just be directly used in the redirect pattern file, e.g.
```
 url(r'^about/(?P<subpath>.*)', lambda req, path: localized_redirect(req, path, 'who-we-are')),
```